stetho/src/main/java/com/facebook/stetho/inspector/elements/android/AccessibilityNodeInfoWrapper.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+package com.facebook.stetho.inspector.elements.android;
+
+import android.support.v4.view.ViewCompat;
+import android.support.v4.view.accessibility.AccessibilityNodeInfoCompat;
+import android.view.View;
+import android.view.ViewParent;
+
+public final class AccessibilityNodeInfoWrapper {
+
+  public AccessibilityNodeInfoWrapper() {
```

Comment:
prefer a private constructor, or none at all

private constructor helps to mimic "static classes" in C#/VB. not having an explicitly defined constructor is equivalent to having this public parameterless constructor, but is a bit cleaner for us here